app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -249,8 +252,25 @@ class LuciBuildService {
       retryIf: (Exception e) => e is BuildBucketException,
     );
     for (Response response in batchResponse.responses) {
-      if (response.error?.code != null) {
+      if (response.error?.code != 0) {
         log.warning('BatchResponse error: $response');
+        continue;
+      }
+
+      if (response.scheduleBuild == null) {
+        log.warning('$response does not contain scheduleBuild');
+        continue;
+      }
+
+      final Build scheduleBuild = response.scheduleBuild;
+      // Tags are List<String> so we need to decode to a single int
+      final List<String> checkrunIdStrings = scheduleBuild.tags['github_checkrun'];
+      final int checkRunId = checkrunIdStrings.map((String id) => int.parse(id)).single;
+      final String buildUrl = 'https://ci.chromium.org/ui/b/${scheduleBuild.id}';
+      // Not all scheduled builds have check runs
+      if (checkRunId != null) {
+        final github.CheckRun checkRun = await githubChecksUtil.getCheckRun(config, slug, checkRunId);
+        await githubChecksUtil.updateCheckRun(config, slug, checkRun, detailsUrl: buildUrl);
```

Comment:
Are these calls fast? They may increase the execution time of this request considerably.

Reply:
I'm not sure. We already run one set of these to create the checks in this handler (ideally this would be a non-issue when we have https://github.com/flutter/flutter/issues/81579). Worst case it'll time out on this nice-to-have functionality